core-management/src/main/java/org/wildfly/extension/core/management/logging/CoreManagementLogger.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2011, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.wildfly.extension.core.management.logging;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * Date: 05.11.2011
+ *
+ * @author <a href="mailto:jperkins@redhat.com">James R. Perkins</a>
+ * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
+ * @author <a href="mailto:david.lloyd@redhat.com">David M. Lloyd</a>
+ * @author <a href="mailto:bgaisford@punagroup.com">Brandon Gaisford</a>
+ */
+@MessageLogger(projectCode = "WFLYCM", length = 4)
+public interface CoreManagementLogger extends BasicLogger {
+
+    /**
+     * A logger with a category of the package name.
+     */
+    CoreManagementLogger ROOT_LOGGER = Logger.getMessageLogger(CoreManagementLogger.class, "org.wildfly.extension.core.management");
+
+    @Message(id = 1, value = "The resource %s wasn't working properly and has been removed.")
+    String removedOutOfOrderResource(final String address);
```

Comment:
Can you change the name of this to 'removedBrokenResource' or something like that?

As I was scanning this PR I interpreted 'out of order' as meaning 'incorrect order' and not as meaning 'broken' and that confused me. No big deal but it may confuse some poor guy maintaining this in 5 years. ;)